### ISSUE TITLE
Jterator dependency problem in 0.2.0

### ISSUE THREAD

**Issue by jluethi** [pelkmanslab/TissueMAPS-OLD#25]
When trying to run a measure_intensity or a measure_morphology module in jterator in the 0.2.0 release (on the http://130.60.24.212/ machine), I run into a missing dependency: These jterator modules require a module called "pysal". Was that not needed before? Or is the dependency just not part of the automatic installer scripts?

![screen shot 2017-02-16 at 15 08 28](https://cloud.githubusercontent.com/assets/18033446/23024351/393add5e-f45a-11e6-881c-1129c0acfbba.png)

**Comment by scottberry** [comment]
good to make a note of this. you could try to ssh to 130.60.24.212 and `pip install pysal` to see if this fixes the problem.

**Comment by jluethi** [comment]
Yes, pip install pysal solved the problem. Alex did this an hour ago and now those modules are fine.
@HackerMD This should also be included in the setup installation, I assume

**Comment by hackermd** [comment]
This is weird. I don't have `pysal` installed on any of my machines and don't have this problem. Have you checked out the master branch of JtModules or are there any new modules?

**Comment by hackermd** [comment]
This got accidentally introduced with TissueMAPS/JtLibrary@6a6760ade9b30ac55638b460dad6a9e59ebc05ca. The package is not required. I removed it with TissueMAPS/JtLibrary@d1433df3f1c59354c58934505fda5955a768f9ec
